src/GitHub/Data/PullRequests.hs

```diff
@@ -31,11 +31,11 @@ data SimplePullRequest = SimplePullRequest
     , simplePullRequestCreatedAt :: !UTCTime
     , simplePullRequestUser      :: !SimpleUser
     , simplePullRequestPatchUrl  :: !URL
-    , simpleIssueState     :: !IssueState
+    , simplePullRequestState     :: !IssueState
```

Comment:
reverted, I was working from a dirty working dir, so I assumed it was a missed ctrl c/v